analysis/SConstruct

```diff
@@ -25,17 +24,25 @@ env = Environment(ENV = {'PATH': os.environ['PATH']},
                               # 'BuildStata' : Builder(action = gs.build_stata),
                               })
 
-
 # Only computes hash if time-stamp changed
 env.Decider('MD5-timestamp') 
 # Extensions to be used when scanning for source files in BuildLyx.
 env.EXTENSIONS = ['.eps', '.pdf', '.lyx']
 SourceFileScanner.add_scanner('.lyx', Scanner(gs.misc.lyx_scan, recursive = True))
-# Load paths
-env['PATHS'] = yaml.load(open('config_global.yaml', 'rU'))
 # Load Stata executable if Stata builder is defined
 if 'BuildStata' in env['BUILDERS'].keys():
-   env['stata_executable'] = misc.load_yaml_value('config_user.yaml', 'stata_executable')
+    env['stata_executable'] = gs.misc.load_yaml_value('config_user.yaml', 'stata_executable')
+# Load global and user config file, record contents of their external directories
```

Comment:
This is a lot of code to put in `SConstruct`. Can we abstract this out and then pass in arguments with `['global', 'user']` and `external` as defaults?

Reply:
I agree that it's a lot of code. I'm not sure where to abstract it to. I originally had it as part of record_dir, but that's no longer appropriate. I've dumped it as two new blocks in configuration.py. That seems like a reasonable resting place for it. Let me know though.